backend/src/services/game.service.ts

```diff
@@ -97,3 +108,43 @@ export const allocatePlayerPunchlines = async (
 
   throw new ServiceError(ErrorType.playerId, "Player does not exist");
 };
+export const shuffleDiscardedSetups = async (game: Game): Promise<void> => {
+  if (game.setups.length <= 5) {
+    const discardedSetups: Setup[] = game.discardedSetups;
+    game.setups.push(...shuffle(discardedSetups));
+    game.discardedSetups.remove(...discardedSetups);
+    await game.save();
+  }
+};
+export const shuffleDiscardedPunchlines = async (
+  game: Game,
+  maxPlayers: number = MaxPlayers
+): Promise<void> => {
+  // Get the next card in the setups pile
+  const topSetup: Setup = game.setups[game.setups.length - 1];
+
+  // Get the size of the first player in the lobby (should be representative of all players)
+  const handSize = game.players[0].punchlines.length;
+
+  let requiredDeckSize: number;
+  // In the case of draw2pick 3, players may need to draw up to 4 cards
+  if (topSetup.type === SetupType.drawTwoPickThree) {
+    requiredDeckSize = handSize === 8 ? 4 * maxPlayers : 3 * maxPlayers;
+  } else {
+    requiredDeckSize = handSize === 8 ? 2 * maxPlayers : maxPlayers;
+  }
+
+  console.log(`req deck: ${requiredDeckSize}`);
```

Comment:
Just get rid of this log 👌🏼

Reply:
Haha must've missed that when I went through and got rid of my logging